CHANGELOG.md

```diff
@@ -1,52 +1,82 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [12.0.0] - 2021-07-07
+
```

Comment:
Where did all of these extra newlines come from :thinking: This would also fail validation.

Reply:
It came from prettier... i have to keep a .prettierignore file without committing it or its gets really opionated about .md files

Reply:
should be fixed 😓

Reply:
Huh, interesting. It's omitted in `yarn lint` explicitly, but we don't have any ignore file committed.... maybe that's something we should consider :thinking: For anyone that runs prettier directly.

Reply:
I think prettier plugin for vscode respects eslintrc, so if were to somehow move that setting into the .eslintrc it should be fine. Otherwise a prettierignore file is also good.